src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -2,18 +2,13 @@
 
 .UserAvatar {
   position: relative;
-  z-index: z(global-profile-icon);
   width: 25px;
   height: 25px;
 
   &:hover {
-    z-index: z(global-profile-icon--hover);
```

Comment:
If these are not used anywhere else, please remove them.

Reply:
One is used in Convers. space
![image](https://user-images.githubusercontent.com/63194656/125317174-79285d80-e341-11eb-81b4-898a35561e95.png)

Removed the other in https://github.com/sparkletown/sparkle/pull/1808/commits/b028120d2984ad6df18f4a3167cb10712e545ab3